CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 
 * Remove deprecated `Kafka.spec.topicOperator` classes and deployment logic
 * Use Java 11 as the Java runtime
+* Remove deprecated `Kafka.spec.topicOperator` classes and deployment logic 
```

Comment:
I think this is here twice now - probably some error in rebasing or something.

Reply:
Fixed.